accounts/forms.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib.auth.forms import UserChangeForm
+from django.contrib.auth.models import User
+from django import forms
+
+
+class editProfileForm(UserChangeForm):
```

Comment:
Where are you using this form class?

Reply:
All of the above are supposed to be in other branch and uploaded by mistke I'll fix it right a way

Reply:
Deleted